src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -274,6 +274,11 @@ public boolean isInstanceValid() {
         return sid != null && instance != null && !sid.isEmpty() && !instance.isEmpty();
     }
 
+    @Override
```

Comment:
What is this overriding from? does this have to come alongside a DarkBotAPI change i'm missing?

Reply:
you are right then I ask for the change in the api

Reply:
The change in API, makes this sid status a String, i believe string is better, just return the result of `sidStat()`, which is either "--", "ERR", "OK" or "KO"

Reply:
updated